### PR TITLE
chore: Add DIRECT_URL to the .env.example file

### PR DESCRIPTION
**Pull request by JayaKrishnaNamburu**
## Description

Connecting to database now needs `DIRECT_URL` too, referene
https://github.com/webstudio-is/webstudio/blob/main/packages/prisma-client/prisma/schema.prisma#L13-L17

```sh
datasource db {
  provider  = "postgres"
  url       = env("DATABASE_URL")
  directUrl = env("DIRECT_URL")
}
```

## Code Review

- [ ] hi @istarkov , I need you to do
  - detailed review (read every line)
  
## Before requesting a review

- [x] made a self-review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
     "storybook": "pnpm run --parallel --recursive storybook:dev",
     "storybook:build": "storybook build",
-    "migrations": "cd apps/builder && pnpm migrations --dev",
+    "migrations": "cd apps/builder && pnpm migrations migrate --dev",
```
Comment (JayaKrishnaNamburu) [inline package.json:17]: When i run migrations in local, it needs either `--dev` or `--force` flag as mandatory. I am manually adding when i run migrations till now.

Comment (kof) [reply package.json:17]: @istarkov

**Comment by kof** [comment]
also its weird that locally user needs both  DATABASE_URL and DIRECT_URL to work, it should be possible to use one variable not 2, I don't really understand how we ended up having both of them

File: package.json (hunk 1 above)
Comment (istarkov) [reply package.json:17]: Prev command already had --dev. nothing has changed except you add `migration`

Comment (JayaKrishnaNamburu) [reply package.json:17]: Apologies, I mixed up. It was the missing _migrate_ command itself

**Comment by istarkov** [comment]
It's 2 different variables on all envs except local. Why we need to do something special only for local development just to reduce one line in local environment.

File: package.json (hunk 1 above)
Comment (istarkov) [reply package.json:17]: we use it not only with migrate ;)

Comment (istarkov) [reply package.json:17]: the only reason it exists is —dev flag ;)

**Comment by istarkov** [comment]
Anyway some env providers, like infisical allows to interpolate env variables like
DIRECT=${DATABASE} 
that simplifies setup of secrets as all variables names exists in all envs (just some interpolated some are not)
infisical ui shows env diffs and absence of some variable will cause even more questions

**Comment by kof** [comment]
> It's 2 different variables on all envs except local. Why we need to do something special only for local development just to reduce one line in local environment.

because its confusing when you just want to quickly get it up and running to contribute, we should fallback to one variable and only require one variable locally for simplicity

Do we need 2 of them in the cloud? or is only one needed depending on how we connect? is it possible to define the one that is being needed?

Can we name them semantically distinct so that I can understand from the naming which one is for which purpose?

**Comment by kof** [comment]
as for more distinct naming, this should work:

url       = env("DATABASE_URL")
directUrl = env("DATABASE_DIRECT_URL")

**Comment by kof** [comment]
fixing that naming in prisma https://github.com/prisma/docs/pull/5612

**Review by kof — APPROVED** [review]

**Pull request merged**